en/guide/routing.md

```diff
@@ -354,3 +354,10 @@ app.use('/birds', birds)
 ```
 
 The app will now be able to handle requests to `/birds` and `/birds/about`, as well as call the `timeLog` middleware function that is specific to the route.
+
+But if the parent rout `/birds` has path params, it will not be accessible by default from the sub-routes,
```

Comment:
Spelling: "rout" -> "route"

Please spell out "parameters".

Comma-splice:
"by default from the sub-routes, to make it..."

s/be: "by default from the sub-routes. To make it..."

Reply:
Thank you for the review, I fixed it.